packages/core/admin/admin/src/utils/getFetchClient.ts

```diff
@@ -71,31 +70,31 @@ const hasProtocol = (url: string) => new RegExp('^(?:[a-z+]+:)?//', 'i').test(ur
 const normalizeUrl = (url: string) => (hasProtocol(url) ? url : addPrependingSlash(url));
 
 type FetchClient = {
-  get: <TData = any, R = AxiosResponse<TData>, TSend = any>(
+  get: <TData = unknown, R = FetchResponse<TData>>(
```

Comment:
How come you want to change it to `unknown` over `any`? I think whilst yeah, `any` is more insecure, it's easier to manage migrations for individuals and most data-fetching libraries prefer it for the flexibility.